app/views/qing_groups/_group.html.erb

```diff
@@ -0,0 +1,46 @@
+         <li class="form-group form-item">
+          <%#= debug @qing_group %>
+          <%#= debug @qing_groups %>
+
+            <div class="form-item-content">
+
+              <%# Standard group? %>
+              <i class="standardized fa fa-certificate"></i>
+              <%# end %>
+
+              <%# Question type icon %>
+              <div class="type">
+                <%= form_item_icon('group') %>
+              </div>
+
+              <div class="base-info">
+
+                <%# Group text %>
+                <span class="name">Name<%#= qing_group.group_name_translations %></span>
+
+                <%# Tags %>
+                <span class="token-input-token-elmo">Tag<%#= tag.name %></span>
+              </div>
+
+            <%# Edit and delete %>
+              <div class="actions">
+
+                <%= link_to content_tag(:i, '', class: 'fa fa-pencil'), 'test' %>
+
+                <%# TODO: replace with shorter delete for group %>
+
+                <%# TODO: determine what happens to items inside group on delete %>
```

Comment:
For now I think let's just forbid deleting a group that is not empty. Which means the trash should not be shown if there are children. Which also means it should appear/disappear if it becomes/stops being empty. This will be tricky. Please add to your todo list for now.